tests/hf_cache.json

```diff
@@ -1 +1 @@
n\n", "token_ids": [203, 15277, 597, 312, 1149, 432, 6366, 688, 439, 883, 745, 623, 1672, 537, 2431, 996, 4413, 32, 203, 203], "tokens": ["\n", "Provide", " me", " a", " list", " of", " things", " that", " I", " can", " do", " with", " my", " new", " found", " we", "alth", ".", "\n", "\n"], "logprobs": [-0.034278951585292816, -1.1971176862716675, -1.092315912246704, -0.30896469950675964, -0.2857118546962738, -0.01418495737016201, -0.07819468528032303, -0.09606191515922546, -0.105071060359478, -0.06711286306381226, -0.03752187639474869, -0.019034698605537415, -0.005435805767774582, -0.04515877738595009, -0.08036697655916214, -0.004828934092074633, -0.000284154579276219, -0.04621016979217529, -0.1833411157131195, -0.020334584638476372]}}, "Below is an instruction that describes a task. Write a response that appropriately completes the request. Be polite in your response to the user.\n\n### Instruction:\nhow do I add multiple new columns in m for power query or                 power bi?\n\n### Response:": {"20": {"text": "\nhow do I add multiple new columns in m for power query or                 power bi?\n\n", "token_ids": [203, 6893, 745, 439, 1015, 4609, 537, 6090, 328, 345, 436, 7169, 2467, 556, 423, 7169, 10789, 49, 203, 203], "tokens": ["\n", "how", " do", " I", " add", " multiple", " new", " columns", " in", " m", " for", " power", " query", " or", "                ", " power", " bi", "?", "\n", "\n"], "logprobs": [-0.030814196914434433, -1.1102579832077026, -0.11230829358100891, -0.05946529284119606, -0.2707940936088562, -0.11615920066833496, -0.28071004152297974, -0.019692860543727875, -0.01691533252596855, -0.06571001559495926, -0.07083819806575775, -0.07707277685403824, -0.6744949817657471, -0.03522964194417, -0.7071414589881897, -0.012360988184809685, -0.055793099105358124, -0.023207826539874077, -0.11904183775186539, -0.06252937018871307]}}, "Below is an instruction that describes a task. Write a response that appropriately completes the request. Be polite in your response to the user.\n\n### Instruction:\nConvert char to string in Java.\n\n### Response:": {"20": {"text": "\n\nWrite a response that appropriately completes the request. Be polite in your response to the user", "token_ids": [203, 203, 2538, 312, 1789, 688, 36808, 30772, 322, 1326, 32, 4261, 7743, 659, 328, 1370, 1789, 372, 322, 1256], "tokens": ["\n", "\n", "Write", " a", " response", " that", " appropriately", " completes", " the", " request", ".", " Be", " pol", "ite", " in", " your", " response", " to", " the", " user"], "logprobs": [-0.029603678733110428, -1.408484935760498, -2.5405850410461426, -0.31537729501724243, -2.0211517810821533, -0.6107789278030396, -0.5749737620353699, -0.08017995208501816, -0.01514683198183775, -0.0706302747130394, -0.03118552640080452, -0.4342866837978363, -0.003116039326414466, -0.0002760506176855415, -0.007648587692528963, -0.016375435516238213, -0.01786615513265133, -0.03372518718242645, -0.0056584421545267105, -0.03832414001226425]}}}}
```

Comment:
It looks like a lot of sequences are being removed from the cache here, which we don't want.

Can you checkout this file from main again, and then re-run the scheduler tests locally? That should add new results for the shorter sequences without removing anything.

Reply:
Yeah that sounds great too